src/svg-renderer.js

```diff
@@ -51,6 +50,25 @@ class SvgRenderer {
          * @type {boolean}
          */
         this.loaded = false;
+
+        /**
+         * The rotation center of the currently loaded SVG, as adjusted to ensure proper subpixel positioning.
+         * Relative to the top-left corner of the adjusted viewbox.
+         * Null when an SVG is not loaded.
+         * @type {?Array<number>}
+         */
+        this.adjustedRotationCenter = null;
```

Comment:
Let's think about how we can move the very scratch-render-api specific functions back into Render to make sure that other dependencies aren't accidentally affected